PendingReleaseNotes.md

```diff
@@ -8,3 +8,5 @@
 ## Features
 
 ## NOTE
+
+- set-metadata flag has been set to true by default.
```

Comment:
Please update the title to point to next release and remove the breaking changes as its already part of the release

Reply:
write `--setmetadata` instead of "set-metadata"